collector/collector.go

```diff
@@ -27,8 +27,10 @@ import (
 	"sync"
 	"time"
 
+	"github.com/google/gopacket/layers"
```

Comment:
This import should follow after the uber arachne imports with one line of separation from latter.